spec/features/user_registers_spec.rb

```diff
@@ -1,12 +1,55 @@
 require 'rails_helper'
 
-feature 'User registers' do
-  scenario 'User registers without invite' do
+feature 'User registers without invite', { js: true, vcr: true} do
```

Comment:
You can also do this differently like:

`feature 'User registers without invite', :js, :vcr do`